src/movies.js

```diff
@@ -1,22 +1,86 @@
 // Iteration 1: All directors? - Get the array of all directors.
 // _Bonus_: It seems some of the directors had directed multiple movies so they will pop up multiple times in the array of directors.
 // How could you "clean" a bit this array and make it unified (without duplicates)?
-function getAllDirectors(moviesArray) {}
+
+function getAllDirectors(moviesArray) {
+
+      const newMovies = moviesArray.map(movie => movie.director);
+      return newMovies;
+  }
```

Comment:
Esto esta perfect!!!

Te lo refactorizo porque lo que queremos es que vayais tirando a esto pero recalco esta perfecto!!

```suggestion
function getAllDirectors(moviesArray) {
    return moviesArray.map(movie => movie.director);
}
```